azure-functions/src/send_grid/message_builder.rs

```diff
@@ -521,11 +578,16 @@ impl MessageBuilder {
     /// # Examples
     ///
     /// ```rust
-    /// # use azure_functions::send_grid::MessageBuilder;
-    /// let message = MessageBuilder::new().attachment("hello.txt", "text/plain", "aGVsbG8gd29ybGQ=").build();
+    /// use azure_functions::bindings::SendGridMessage;
+    ///
+    // @peterhuene I changed the content here because when I was reading through and saw the original string
```

Comment:
The SendGrid API uses base64 encoding for the attachment content.

Let's change the example back, but add a note in the method's documentation that attachment content is expected to be base64 encoded.

Reply:
Ah, I see. Is it worth adding the `base64` crate to the project and allowing the user to input plaintext that is encoded in the `attachment` function? Does it take the implementation too far away from the C# implementation?

Reply:
I think calling out the fact it expects base64 data would suffice, rather than taking a slice and doing the encoding ourselves, at least to make it match the API I was copying.